services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -1253,8 +1281,53 @@ def delete(self, commit=True):
             doc.mine_document.delete(False)
         return super(ProjectSummary, self).delete(commit)
 
-    def send_project_summary_email(self, mine):
-        emli_recipients = PROJECT_SUMMARY_EMAILS
+    def send_project_summary_document_email(self, mine) -> None:
+        if is_feature_enabled(Feature.MINE_APPLICATION_FILE_UDPATE_ALERTS):
+            message = f'File(s) in project {self.project.project_title} has been updated for mine {mine.mine_name}'
+            project_lead_email = self.project_lead_email
+
+            emails = {
+                'SUB': [PERM_RECL_EMAIL],
+                'ASG': [PERM_RECL_EMAIL, project_lead_email],
+                'CHR': [PERM_RECL_EMAIL, project_lead_email] 
+            }
+            email_recipients = emails.get(self.status_code)
+
+            if email_recipients is not None:
+                emli_body = open("app/templates/email/projects/emli_project_summary_email.html", "r").read()
+                subject = f'Project Description Documents Notification for {mine.mine_name}'
+                cc = [MDS_EMAIL]
+
+                emli_context = {
+                    "project_summary": {
+                        "project_summary_description": self.project_summary_description,
+                    },
+                    "mine": {
+                        "mine_name": mine.mine_name,
+                        "mine_no": mine.mine_no,
+                    },
+                    "message": message,
+                    "core_project_summary_link": f'{Config.CORE_WEB_URL}/pre-applications/{self.project.project_guid}/overview'
+                }
+                EmailService.send_template_email(subject, email_recipients, emli_body, emli_context, cc=cc)
+
+
+    def send_project_summary_email(self, mine, message):
+        current_app.logger.info(f'HI TARA Sending email with message {message}')
```

Comment:
Is this log meant to be removed?

Reply:
Oops, I thought I caught all of these! Will remove.